### DISCUSSION TITLE
2022 年年终总结投资篇 - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#30] [Announcements]
# 2022 年年终总结投资篇 - forecho's Blog

引言
2022 这操蛋的一年终于要结束了，按照惯例，每年来一次投资年终总结，来看看这一年的投资表现如何。这是这系列的第四篇文章，前三篇文章：

【深度理财 34】2021 年年终总结投资篇
【深度理财 19】2020 年年终总结投资篇
【深度理财 06】聊聊这些年我买过的股票

加一个风险提示：本篇文章仅代表我的个人观点，不构成与其他人相关的任何投资建议。在作出任何投资决定前，投资者应根据自身情况考虑投资产品相关的风险因素，并于需要时咨询专业投资顾问意见。

基金
概况

今年基金亏惨了，总体收益 -13% 左右，去年是 +6%，去年的利润全没了还不说，还倒亏钱了。
因为今天中国的疫情封控，导致经济下滑，股市自然好不到哪里去了，我今年主要的仓位：

新能源（40%）
医疗（30%）
白酒（20%）
其他（10%）

https://blog.forecho.com/2022-review-investment.html

**Comment by yaoyuanyy** [comment]
楼主如何注册登录的Telegram，电脑客户端无法注册登录了，你是如何解决的

**Reply by forecho** [reply]
我注册早，如果国内手机号收不到验证码的话，试试买一个 GV 号注册。
